packages/shared/src/components/ProfilePicture.tsx

```diff
@@ -7,6 +7,7 @@ import { Image, ImageType } from './image/Image';
 import { useRequestProtocol } from '../hooks/useRequestProtocol';
 
 export enum ProfileImageSize {
+  Size16 = 'size16',
```

Comment:
🙈 this didn't fit in any of the ones we have already defined?

Reply:
Nope super tiny format this one.